src/SamplesApp/UITests.Shared/Helpers/BindableBase.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Runtime.CompilerServices;
+using System.Text;
+
+namespace UITests.Shared.Helpers
+{
+	[Windows.UI.Xaml.Data.Bindable]
+	public class BindableBase : INotifyPropertyChanged
```

Comment:
You may use `ViewModelBase` for that ;-)

Reply:
In this case, the class is more lightweight (there is more additional logic in `ViewModelBase`) and`ViewModelBase` requires passing `Dispatcher` in the constructor, so for these reasons it seemed useful to have an alternative. But can refactor to `ViewModelBase` if you prefer :-) .

Reply:
It's not required.

Reply:
I will keep it for now then 😉